src/PrestaShopBundle/Routing/Exception/RouteNotFoundException.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Routing\Exception;
+
+use PrestaShop\PrestaShop\Core\Exception\ErrorMessageException;
+use Throwable;
```

Comment:
As found by sarjon in https://github.com/PrestaShop/PrestaShop/pull/10865 : careful with `Throwable` we still have to be php5.6-compatible

Reply:
ok, I integrated the modif but can you check if it is good `TranslatableCoreException`
Especially the `use` part, is it ok to have `use Throwable` in php 5.6 or should I use `\Throwable` and `\Exception` instead?

Reply:
I tried, it seems the `use` is not a issue for php5.6